src/App.js

```diff
@@ -1,8 +1,23 @@
 import React from "react";
+import NavBar from "./Components/NavBar";
+import UserProfile from "./Components/UserProfile";
+import Post from "./Components/Post";
+import Posts from "./Components/Posts";
+import Contacts from "./Components/Contacts";
+import ContactUserCard from "./Components/ContactUserCard";
 import "./index.css";
 
 const App = () => {
-  return <p>Hello, world!</p>;
+  return (
+    <div>
+      <NavBar />
+      <UserProfile />
+      <Post />
```

Comment:
Hey James, nice job! One quick comment: I think you may realize this now from all the work we've done over the last week, but in your <App /> component, you have <Post /> and <Posts />, along with Contacts and ContactUserCard. 'Post' should be imported and used within the 'Posts" component, as well as 'ContactUserCard' being imported and used within the 'Contacts' component.